ext/nginx.conf

```diff
@@ -10,44 +10,43 @@ upstream bancho {
 # a.ppy.sh is used for osu! avatars
 
 server {
-	listen 443 ssl;
+	listen 80;
 
 	# XXX: you'll need to edit this to match your domain (mine's cmyui.xyz)
-	server_name ~^(?:c[e4]?|osu|a|b|api)\.cmyui\.xyz$;
-
-	# XXX: you'll need to edit this to match your certificate and key
-	ssl_certificate     /etc/letsencrypt/live/cmyui.xyz/fullchain.pem;
-	ssl_certificate_key /etc/letsencrypt/live/cmyui.xyz/privkey.pem;
-
-	# TODO: further ssl configuration
-	ssl_ciphers "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH:@SECLEVEL=1";
+	server_name ~^(?:c[e4]?|osu|b|api)\.your\.domain$;
 
 	location / {
 		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 		proxy_set_header X-Real-IP  $remote_addr;
 		proxy_set_header Host $http_host;
 		add_header Access-Control-Allow-Origin *;
 		proxy_redirect off;
-		proxy_pass http://bancho;
+		proxy_pass http://bancho_server;
 	}
 }
 
 server {
-	listen 443 ssl;
-
+	listen 80;
 	# XXX: you'll need to edit this to match your domain (mine's cmyui.xyz)
-	server_name assets.cmyui.xyz;
+	server_name assets.your.domain;
 
-	# XXX: you'll need to edit this to match your certificate and key
-	ssl_certificate     /etc/letsencrypt/live/cmyui.xyz/fullchain.pem;
```

Comment:
Why would you refactor this? It's great example of how path should look and help enduser spent less time in google.